addr_linux.go

```diff
@@ -69,6 +74,7 @@ func AddrDel(link Link, addr *Addr) error {
 //
 // If `addr` is an IPv4 address and the broadcast address is not given, it
 // will be automatically computed based on the IP mask if /30 or larger.
+// If `net.IPv4zero` is given as the broadcast address, broadcast is disabled.
```

Comment:
Does it make sense to specify this for a delete address operation?

Reply:
No, it doesn't. I only added it for consistency with the other comments. It's added in this [commit](https://github.com/vishvananda/netlink/commit/aad0baef2856c220acdcb053b1d3d4391d8afd7b).

Reply:
Maybe I should remove the part about broadcast auto computation as well.